Update hackathon page (#1245)
nf-core/nf-co.re: markdown/events/2022/hackathon-october-2022.md

Please join us for the October 2022 nf-core hackathon!

This hackathon will be held in advance of the _**Nextflow Summit 2022**_ in Barcelona, Spain 🇪🇸. You can find out more information about the summit at <https://summit.nextflow.io>.

In addition to the in-person event, we will endeavour to run a hybrid option online in [gather.town](https://gather.town/).

> Presentations will be run using Zoom and live-streamed to YouTube,

Author: I took that from past hackathons. Will they happen only in gather town and won't be recorded to post on YouTube after the hackathon?

> Presentations will be run using Zoom and live-streamed to YouTube,

Reviewer: Oh I'm not sure now.. 🤔 Maybe needs discussion, can take it in Slack.

This group will focus on all kind of documentation, from pipelines themselves to modules and general documentation to aid in getting started with nf-core and Nextflow. We are also planning on making introduction videos for each pipeline.

>   - [<i class="fab fa-slack"></i> `#hackathon-oct-2022-documentation`](https://nfcore.slack.com/archives/C041QU9SX09)

Reviewer: Are these indented for a reason? It's breaking prettier linting..

Author: They pass prettier linting in my computer 🤔 but I will remove the indentation just in case

We will be monitoring the status of the pandemic as the event draws closer and reserve the right to make changes in our policies if we deem that to be necessary.

# Safety Information at the Hackathon

Please note that our [Code of Conduct](https://nf-co.re/code_of_conduct) applies to the Hackathon, and all participants need to abide by our guidelines to participate. We should all feel responsible for making nf-core events safe and fun for everyone.

In addition, please respect the following at all times during the hackathon:

- Do not take screenshots of groups you are working within without asking for consent of all individuals in the group. Remember, not everyone wants their photographs shared on social media.

## Reporting CoC Violations during the hackathon

In case of an immediate perceived threat at the hackathon, please reach out to any of the following individuals on Slack. Their names and their location (online/onsite) are indicated.

> - Chris Hakkaart (onsite)

Reviewer: ```suggestion
- Cris Tuñí (onsite)
```

Author: oops 😇

You can add your community event using [this link](https://seqera.typeform.com/streaming-event).

## Social Activities

During the hackathon, we will have a few light-hearted fun and games!

### On-site

> - Monday evening: Beers at Seqera Office!

Reviewer: ```suggestion
- Monday evening: Beers and pizza (non-alcoholic options available) at the Seqera Office!
```

- Tuesday evening: Dinner!

- The traditional sock hunting in Gathertown will also be held in person! Take pictures of as many socks as you can find.

> - Finally, on Thursday, we will be running a short quiz!

Author: Definitely not Thursday, sorry. We didn't say a day, should we say Wednesday?

> - Finally, on Thursday, we will be running a short quiz!

Reviewer: Wednesday will be a little short is the only thing, as we're stopping at lunch time.

- Finally, on Thursday, we will be running a short quiz!

All social activities are of course optional, but hope to see as many people joining in as possible :tada:

# COVID control

Please remember that the COVID-19 pandemic is not over.
If you intend on joining the hackathon in person, we request that you get vaccinated and tested, if possible. We ask for anyone who displays symptoms to not mingle with other participants. Whilst it is not feasible for us to enforce any rules, we request that attendees behave responsibly.

We will be monitoring the status of the pandemic as the event draws closer and reserve the right to make changes in our policies if we deem that to be necessary.

# Safety Information at the Hackathon

Please note that our [Code of Conduct](https://nf-co.re/code_of_conduct) applies to the Hackathon, and all participants need to abide by our guidelines to participate. We should all feel responsible for making nf-core events safe and fun for everyone.

In addition, please respect the following at all times during the hackathon:

> - **Never share the jitsi and zoom links** for the hackathon on social media.

Reviewer: Ooh this is an old one! 😅

> - Do not take screenshots of groups you are working within without asking for consent of all individuals in the group. Remember, not everyone wants their photographs shared on social media. Not respecting the latter amounts to doxing, which is also in direct violation of the nf-core [Code of Conduct](https://nf-co.re/code_of_conduct).

Reviewer: ```suggestion
- Do not take screenshots of groups you are working within without asking for consent of all individuals in the group. Remember, not everyone wants their photographs shared on social media.
```

## Reporting CoC Violations during the hackathon

In case of an immediate perceived threat at the hackathon, please reach out to any of the following individuals on Slack. Their names and the time-zones they are located in are indicated.

> - Alexander Peltzer (CET)

Reviewer: Should just be the 3x safety team. No mention of time zones, just say online / in person.

- Harshil Patel (GMT)
- Phil Ewels (CET)
- Gisela Gabernet (CET)
- James A. Fellows Yates (CET)
- Matthias Hörtenhuber (CET)
- Maxime Garcia (CET)
- Renuka Kudva (CET)
- Kevin Menden (CET)
- Friederike Hanssen (CET)
- Michael Heuer (PST)

> You can also report any CoC violations directly to safety [at] nf-co [dot] re. Our safety officer, Renuka Kudva, will contact you to follow up on your report.

Reviewer: ```suggestion
You can also report any CoC violations directly to safety [at] nf-co [dot] re. Our safety officers will contact you to follow up on your report.
```

> In your email report, please do your best to include:

Reviewer: Not sure that we need this level of detail tbh

- Your contact information.
- Identifying information (e.g. names, nicknames, pseudonyms) of the participant who has violated the Code of Conduct.
- The behaviour that was in violation and the circumstances surrounding the incident.
- The approximate time of the behaviour (if different than the time the report was made)
- Other people involved in the incident if applicable.
- If you believe the incident is ongoing, please let us know.
- If there is a publicly available record (e.g. mailing list record, a screenshot - please attach if so).
- Any additional helpful information.

> After you file a report, uur safety officer, Renuka Kudva, will contact you to follow up on your report.

Reviewer: ```suggestion
```

> ### Who will read and handle reports?

Reviewer: This stuff is all in the CoC already right? I think we can delete from here onwards..
